TravisBot/AppDelegate.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  AppDelegate.swift
+//  TravisBot
+//
+//  Created by Douglas Garcia Barroso on 22/03/20.
+//  Copyright © 2020 Douglas Garcia Barroso. All rights reserved.
+//
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/dbgarcia/danger-bot/blob/4ad3f65193926f1211a9cfb888b46cbbba684b48/TravisBot/AppDelegate.swift#L16"></span>Line should be 130 characters or less: currently 145 characters<br />
<code>line_length</code> <code>AppDelegate.swift:16</code></del></td>
    </tr>
  </tbody>
</table>